py2-memory_profiler.spec

```diff
@@ -0,0 +1,16 @@
+### RPM external py2-memory_profiler 0.52.0
```

Comment:
Look at what I found :-)
@ticoann did you create this spec from scratch or you got it from my working area?
I think there was still an issue to be fixed (otherwise it would fail to build), but since we got +1, maybe it worked just fine in the end
...oh, it actually hasn't been built, I guess that's why the +1. Anyways, I'll try to get back to it once I have spare cycles.